pkg/ruler/ruler_test.go

```diff
@@ -6,13 +6,15 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"os"
+	"strings"
 	"sync"
 	"testing"
 	"time"
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"
+	promTU "github.com/prometheus/client_golang/prometheus/testutil"
```

Comment:
minor nit but when we import both testutil packages in prometheus projects we name this one `prom_testutil "github.com/prometheus/client_golang/prometheus/testutil"`

Reply:
Good catch.